test/src/main/java/ai/timefold/solver/test/api/score/stream/MultiConstraintVerification.java

```diff
@@ -9,14 +9,13 @@ public interface MultiConstraintVerification<Solution_> {
      *
      * @param facts at least one
      */
-    // TODO @NonNull Object correct here?
     @NonNull
     MultiConstraintAssertion given(@NonNull Object @NonNull... facts);
 
     /**
      * As defined by {@link SingleConstraintVerification#givenSolution(Object)}.
      */
     @NonNull
-    MultiConstraintAssertion givenSolution(@NonNull Solution_ solution);
+    MultiConstraintListener givenSolution(@NonNull Solution_ solution);
```

Comment:
This is not backwards compatible. The return type must change compatibly.
Therefore the listener (which is a bad name anyway, needs changing) has to extend the assertion.

Reply:
`MultiConstraintListener` already extends `MultiConstraintAssertion`, and it should maintain backward compatibility.